src/Test.EndToEnd.Baselining/Program.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using CommandLine;
+using Test.EndToEnd.Baselining.Options;
+
+namespace Test.EndToEnd.Baselining
+{
+    /// <summary>
+    ///  Test.EndToEnd.Baselining is a console app for conducting end-to-end testing of the baselining algorithm.
+    ///  It uses sample SARIF logs, some of which contain sensitive data, so the test content is not in this repository.
```

Comment:
>repository [](start = 109, length = 10)

IMO that means this whole project belongs in another, private, repo. Even the "Content Folder Structure" below reveals organizational and project details. We shouldn't put a project that only some people can use in a public repo.

Reply:
@michaelcfanning, what do you think about this? I made a separate repo for the test content, and could move the harness there. The harness can also be used with non-private content, if it's here, and it's a bit more discoverable to people who might change the Baselining later (so they'd go looking for the private repo)

Reply:
I *think* might wanted the harness here and just the content elsewhere. This means anyone can run with public test content (once we have some). I can move it to the content repo later if that's what was wanted.